components/gitpod-db/src/typeorm/migration/1662639748206-CostCenterPaymentStrategy.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const D_B_COST_CENTER = "d_b_cost_center";
+const COL_CREATION_TIME = "creationTime";
+const COL_BILLING_STRATEGY = "billingStrategy";
+
+export class CostCenterPaymentStrategy1662639748206 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, D_B_COST_CENTER, COL_CREATION_TIME))) {
+            await queryRunner.query(
+                `ALTER TABLE ${D_B_COST_CENTER} ADD COLUMN ${COL_CREATION_TIME} varchar(30) NOT NULL, ALGORITHM=INPLACE, LOCK=NONE `,
```

Comment:
they will be filled up with the implicit default value for the `varchar` type right? which is `''`, an empty string.

Reply:
Oh, true. That was not my intend, but I think it will not break.

Reply:
I'm actually not sure what MYSQL will do with existing records here 😬 The migration worked well against an empty table.

Reply:
> they will be filled up with the implicit default value for the varchar type right? which is '', an empty string.

Just verified that this is the case, indeed